### PR TITLE
Add a GH Action workflow for continuous delivery

### PR DESCRIPTION
**Pull request by ikalnytskyi**
Add a GitHub Action workflow that is triggered on push to master or
manually that deploys from master branch to production. This is the
first step towards infrastructure as a code. Further enhancements could
be to move production variables from job definition to ansible variables
file.

### CODE REVIEW

File: .github/workflows/cd.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+name: cd
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
```
Comment (ikalnytskyi) [inline .github/workflows/cd.yaml]: This is temporary. I'll remove it before merging. This is just to test this workflow on this PR.

File: .github/workflows/cd.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    environment:
+      name: xsnippet.org
+      url: https://xsnippet.org
+    concurrency: xsnippet.org
```
Comment (ikalnytskyi) [inline .github/workflows/cd.yaml:14]: Only one job at a time (even if it will be triggered concurrently).

Comment (malor) [reply .github/workflows/cd.yaml:14]: Cool trick!

**Review by malor — DISMISSED** [review]
Nice! Thank you!

**Review by malor — APPROVED** [review]

**Pull request merged**